pull-request/src/person.js

```diff
@@ -12,4 +12,5 @@ Person.prototype.fullname = function() {
 
 Person.prototype.wishHappyBirthday = function() {
     return "Happy birthday " + this.firstName;
+    this.age++;
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Remove this code after the "return" statement. ([javascript:UnreachableCode](http://localhost:9760/coding_rules#rule_key=javascript%3AUnreachableCode))

[See it in SonarQube](http://localhost:9760/project/issues?id=training%3Apull-request&pullRequest=52&issues=AWl8QWTw1Va9TISLgQ_3&open=AWl8QWTw1Va9TISLgQ_3)

[ISSUE_KEY]: AWl8QWTw1Va9TISLgQ_3